src/syntax/errors/list.rs

```diff
@@ -88,9 +91,24 @@ pub fn get_error_desc(err: &ErrorKind) -> (&'static str, String, &'static str) {
             "",
         ),
         ErrorKind::MissingVariants => (
-            "E0010",
+            "E0011",
             "Expected at least one variant after \"->\".".to_owned(),
             "",
         ),
+        ErrorKind::ForbiddenAttributeExpression => (
```

Comment:
Perhaps rename this to `ForbiddenPrivateAttributeExpression` for consistency with `ForbiddenPublicAttributeExpression `?